### PR TITLE
enable benchmarks in dockerimage & allow reserve_origin to add/reduce…

### PR DESCRIPTION
**Pull request by GopherJ**
… at anytime

### CODE REVIEW

**Comment by yz89** [comment]
why do you want to allow reserve_origin to add/reduce any kinds of currencies?

**Comment by GopherJ** [comment]
> why do you want to allow reserve_origin to add/reduce any kinds of currencies?

because sometimes we may shutdown a market but needs to do operations on reserve I think. Will that be possible?

File: Dockerfile.release [hunk 1]
```diff
@@ -6,7 +6,7 @@ WORKDIR /parallel
 
 COPY . /parallel
 
-RUN cargo build --$PROFILE --bin parallel-dev
+RUN cargo build --$PROFILE --bin parallel-dev --features runtime-benchmarks
```
Comment (kaichaosun) [inline Dockerfile.release:9]: This feature should only be used in benchmark, not the online binary.

Comment (GopherJ) [reply Dockerfile.release:9]: it'll make benchmark on our server easier I think

**Pull request closed without merging**